src/index.html

```diff
@@ -5,8 +5,451 @@
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#">
+          <img 
+            class="header__img" 
+            src="/images/logo.png" 
+            alt="mojo-logo"
+          >
+        </a>
+      </div>      
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a 
+              href="#" 
+              class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#" 
+              class="nav__link">
+              samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#" 
+              class="nav__link">
+              smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#" 
+              data-qa="nav-hover"
+              class="nav__link"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link">
+              gadgets            
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#"
+              class="nav__link">
+              tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#" 
+              class="nav__link">
+              photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="#" 
+              class="nav__link">
+              video
+            </a>
+          </li>
+        </ul>
+      </nav> 
+    </header>
+
+    <main class="page">
+      <div class="container page__container">
+        <section class="card container__card" data-qa="card">        
+          <img 
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="photo of an imac"
+          > 
+        
+          <div class="card__product-title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+
+          <div class="card__sku">
+            Product code: 195434 
+          </div>
+        
+          <div class="review card__review">
+            <div class="star stars--4">
+              <div class="star__star"></div>
+              <div class="star__star"></div>
+              <div class="star__star"></div>
+              <div class="star__star"></div>
+              <div class="star__star"></div>
+            </div>
+            <text class="review__text">
+              Reviews: 5
+            </text>
+          </div>
+
+          <div class="price">
```

Comment:
here this is not the right way to do it
you should introduce another class called `page__price` and could give it external styllings
and use it in HTML like this 
```suggestion
          <div class="price page__price">
```

Reply:
also that class stylling should be done in `page.scss` block

Reply:
<img width="847" alt="Screen Shot 2022-08-16 at 7 43 24 PM" src="https://user-images.githubusercontent.com/108151633/184944585-fa7be8f1-cf45-4b27-be93-404351598593.png">

Lint error message received by adding page__price element: Element isn't inside the parent block  [page__price]    (element-inside-parent-block) 

I do not understand why this is an issue.

Reply:
done. tnx